parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -72,6 +72,12 @@ if [[ "${SHOULD_CONFIGURE_CUSTOM_CA_TRUST}" == "true" ]]; then
     logs_to_events "AKS.CSE.configureCustomCaCertificate" configureCustomCaCertificate || exit $ERR_UPDATE_CA_CERTS
 fi
 
+dns_domain="mcr.microsoft.com"
```

Comment:
nit: dns stands for Domain Name System. So dns_domain seems duplicated to me. My opinion: just `domain` or `domain_name`?